sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/EncryptionBridgeInternal.java

```diff
@@ -12,6 +12,11 @@
 
 import static com.azure.cosmos.implementation.Warning.INTERNAL_USE_ONLY_WARNING;
 
+/**
+ * DO NOT USE.
+ * This is meant to be used only internally as a bridge access to classes in
+ * com.azure.cosmos.encryption
```

Comment:
@srnagar - this is actually correct, these helpers are only used to access classes from public surface - since those classes have non public members to avoid exposing them to public surface. So we don't need to change this documentation.